documentation/library_guide/parallel_api/additional_algorithms.rst

```diff
@@ -74,4 +74,15 @@ header.  All algorithms are implemented in the ``oneapi::dpl`` namespace.
 
     input sequence:  [0, 2, 2, 2, 3, 3, 3, 3, 6, 6]
     search sequence: [0, 2, 4, 7, 6]
-    result sequence: [1, 4, 8, 10, 10]
\ No newline at end of file
+    result sequence: [1, 4, 8, 10, 10]
+
+* ``sort_by_key``: performs a stable key-value sort. The algorithm sorts sequence's keys according to 

```

Comment:
My suggestions:
``sort_by_key``: performs a stable key-value sort. The algorithm sorts the sequence's keys according to a comparison operator. If no comparator is provided, then a function object that uses ``operator<`` to compare the elements is used. The sequence's values are permutated according to the sorted sequence's keys. For correct computation, the number size of a key sequence and the size of a value sequence should be the same. For example::

Reply:
I believe we iterate over the wording several times, so after the latest update we are good to go